tests/playwright-test/playwright-test.config.ts

```diff
@@ -22,9 +22,11 @@ const config: Config = {
   timeout: 30000,
   forbidOnly: !!process.env.CI,
   preserveOutput: process.env.CI ? 'failures-only' : 'always',
-  projects: [
-    { name: 'playwright-test' },
-  ]
+  projects: process.env.PLAYWRIGHT_DOCKER ? [
+    { name: 'vrt', testMatch: ['reporter-html.spec.ts'] },
```

Comment:
I'd instead use `foo.spec.ts` vs `foo.visual.ts` convention or something like that. You don't want to hardcode file names here.

Reply:
done